app/javascript/components/forms/DeleteUserForm.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import {
+  Button,
+} from 'react-bootstrap';
+import Form from './Form';
+import useDeleteUser from '../../hooks/mutations/users/useDeleteUser';
+import { useAuth } from '../../contexts/auth/AuthProvider';
+
+export default function DeleteUserForm() {
+  const currentUser = useAuth();
+  const methods = useForm();
```

Comment:
It's recommended always to provide default values to `react-hook-form`.
Rules section of https://react-hook-form.com/api/useform

Reply:
It has no values for now, the form is only a button.
Will probably have to add a password confirmation field or a checkbox to confirm the deletion eventually.

Reply:
My bad, I mixed between this form and the update profile one 🤣